poker_ai/ai/multiprocess/server.py

```diff
@@ -86,7 +86,7 @@ def search(self):
         progress_bar = progress_bar_manager.counter(
             total=self._n_iterations, desc="Optimisation iterations", unit="iter"
         )
-        for t in range(self._start_timestep, self._n_iterations):
+        for t in range(self._start_timestep, self._n_iterations + 1):
```

Comment:
Updating for dump iteration. If n_iterations is divisible by the dump_int, then we dump the final iteration.